doc/api/inspector.md

```diff
@@ -108,6 +108,49 @@ with an error. [`session.connect()`] will need to be called to be able to send
 messages again. Reconnected session will lose all inspector state, such as
 enabled agents or configured breakpoints.
 
+## inspector.ws
+
+The Inspector can be interacted with via a websocket-based protocol. These APIs
+allow programatic control of the websocket, equivalent to use `--inspect*` CLI
+options.
+
+### ws.listen()
+
+Start the websocket listener.
+
+### ws.unlisten()
+
+Stop the websocket listener.
+
+### ws.id()
+
+Return the UUID string of the listener, or undefined if the listener is not
+started.
+
+### ws.host([newHost])
+
+* host {string} Set a new host to bind to (will not effect a currently
+  started listener). Optional.
+
+Return the currently configured host string.
+
+### ws.port([newPort])
+
+* port {number} Set a new port to bind to (will not effect a currently
+  started listener). Optional.
```

Comment:
right now, nothing, it has no effect (until the listeners is closed and listen is called again).